common/config/rush/pnpmfile.js

```diff
@@ -34,5 +34,9 @@ function readPackage(packageJson, context) {
   //  packageJson.dependencies['log4js'] = '0.6.38';
   // }
 
+  if (packageJson.name === 'acorn-jsx') {
+    delete packageJson.peerDependencies['acorn'];
+  }
```

Comment:
Due to recent updates to acorn-jsx, the carrot version matching now introduces a breaking error during builds. I have reported this to Engineering. This is my temporal solution.